functions/src/modules/pm/categories/updateCategory.ts

```diff
@@ -1,5 +1,80 @@
-import { ExpressFunction } from "../../../@types";
+import type { ExpressFunction } from "../../../@types";
+import { firebaseApp } from "../../../config/firebaseConfig";
+import {
+  expressErrorHandlerFactory,
+  requestBodyTypeValidator,
+} from "../../../helpers";
+import { uasPermissionSwitch } from "../../../systems/uas";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
+const updateCategory: ExpressFunction = (req, res, next) => {
+  uasPermissionSwitch({
+    organizer: { accepted: validateOrganizer },
+    sponsor: { accepted: executeIfSponsorMatches },
+  })(req, res, next);
+};
 
-export default replaceMe;
+const validateOrganizer: ExpressFunction = (req, res, next) => {
+  const validationRules: ValidatorObjectRules = {
+    type: "object",
+    rules: {
+        prizeCategoryName: { rules: ["string"] },
+        prizeCategoryDescription: { rules: ["string"]},
+        eligibility: { rules: ["string"]},
+        approvalStatus: { rules: [{
+          type: "enum",
+          rules: ["string"],
```

Comment:
Same as with create, this needs to contain the actual values.
```suggestion
          rules: ["approved", "rejected", "inProgress", "awaitingApproval"],
```